### PR TITLE
update getting started docs

### PR DESCRIPTION
**Pull request by 04kash**
## Description

Updated the `getting-started.md` file to include changes made to `yarn start` and `yarn dev`

## Which issue(s) does this PR fix

- Fixes #?

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: showcase-docs/getting-started.md [hunk 1]
```diff
@@ -341,7 +341,23 @@ The easiest and fastest method for getting started: Backstage Showcase app, runn
...
+
+To use `yarn dev`, change the following in your app-config.local.yaml:
+
+```yaml
+app:
+  baseUrl: http://localhost:3000
+
+backend:
+  baseUrl: http://localhost:7007
+``cors:
```
Comment (Zaperex) [inline showcase-docs/getting-started.md:354]:
```suggestion
  cors:
```

File: showcase-docs/getting-started.md [hunk 2]
```diff
@@ -341,7 +341,23 @@ The easiest and fastest method for getting started: Backstage Showcase app, runn
...
 7. Navigate to <http://localhost:7007>
 
-8. Alternatively, you can start application using `yarn dev` which would run the frontend using webpack, which allows for hot reloads. This will automatically open `http://localhost:3000` in your default browser.
+8. Alternatively, you can start application using `yarn dev` which would run the frontend using webpack, which allows for hot reloads.
+
+To use `yarn dev`, change the following in your app-config.local.yaml:
+
+```yaml
+app:
+  baseUrl: http://localhost:3000
```
Comment (gashcrumb) [inline showcase-docs/getting-started.md:350]: This should be 7007 when the app is served by app-backend

File: showcase-docs/getting-started.md [hunk 3]
```diff
@@ -341,7 +341,23 @@ The easiest and fastest method for getting started: Backstage Showcase app, runn
...
+
+backend:
+  baseUrl: http://localhost:7007
+  cors:
+    origin: http://localhost:3000
```
Comment (gashcrumb) [inline showcase-docs/getting-started.md:355]: Think this also should be 7007, no?

File: showcase-docs/getting-started.md (hunk 2 above)
Comment (gashcrumb) [reply showcase-docs/getting-started.md:350]: ah, never mind, I should read better :smile:

**Review by JessicaJHee — APPROVED** [review]

**Comment by gashcrumb** [comment]
@kadel @nickboldt wonder if this has to be merged manually as it's not code changes but readme updates

**Comment by rm3l** [comment]
> @kadel @nickboldt wonder if this has to be merged manually as it's not code changes but readme updates

/override "PR Docker Build"

**Pull request merged**